### ISSUE TITLE
A11y_SQL database projects extension_View And More Action_Update Project From Database_Name Role Value: Name is not defined for the "Server", "Data base" and "Location" combo box.

### ISSUE THREAD

**Issue by Samhitha66** [microsoft/azuredatastudio#22100]
### GitHub Tags
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Name Role Value; #WCAG4.1.2; #Win11; #SH-SQL Azure Data Studio-Win32-March23; #Win32; #SQL Azure Data Studio; #NVDA; #Screenreader;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"**

"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Environment Details
Application Name: SQL Azure Data Studio
Version: 1.41.2 (user setup)
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Screen reader: NVDA

### Repro Steps

1. Launch Azure Data Studio
2. Tab till extension and search for “SQL Database Projects” and install it.
3. Tab till database project and press enter. “Database Project” panel will open.
4. Tab till “View and More Action” button and press enter.
5. Tab till “Update Project from Database” and press enter.
6. Tab till "Server", "Data base" and "Location" combo box.
7. Verify whether the name is not defined for the "Server", "Data base" and "Location" combo box or not.

### Actual Result
Name is not defined for the "Server", "Data base" and "Location" combo box.

### Expected Result
Name should be defined for the "Server", "Data base" and "Location" combo box.

### User Impact
Screen reader user will not be able to get the proper information about the "Server", "Data base" and "Location" combo box.

### Attachment
![Presentation2](https://user-images.githubusercontent.com/99243079/222413620-d4dc6a78-ae49-4e7e-9152-4fc1b56b15a8.gif)

**Comment by Samhitha66** [comment]
Regressed the issue in the below environment
Version: 1.43.0-insider (user setup)
Commit: afafee844ce2b07eb55bdb6f40ee1ce88c0bbf4e
Date: 2023-03-29T05:28:54.263Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
![image](https://user-images.githubusercontent.com/99243079/228733493-14a28a66-edab-4260-aad2-8ef5814a2a44.png)
The issue is fixed. Hence closing the bug. Please find the attachment for the reference.
![Bug_22100_Fixed](https://user-images.githubusercontent.com/99243079/228733570-3e24a89e-d317-4a87-ace9-573897f207e3.gif)
